src/spec/product-type-importer.spec.coffee

```diff
@@ -0,0 +1,113 @@
+Promise = require 'bluebird'
+ProductTypeImporter = require '../lib/product-type-import'
+{SphereClient} = require 'sphere-node-sdk'
+
+errMissingCredentials = 'Missing configuration in env variables'
+
+argv =
+  clientId: process.env.SPHERE_CLIENT_ID
+  clientSecret: process.env.SPHERE_CLIENT_SECRET
+  projectKey: process.env.SPHERE_PROJECT_KEY
+  logSilent: true
+
+testProductType = {
+  "name": "top_and_shirts",
+  "description": "Tops & Shirts",
+  "attributes": [
+    {
+      "name": "designer",
+      "label": {
+        "de": "Designer",
+        "en": "designer"
+      },
+      "type": {
+        "name": "text"
+      },
+      "attributeConstraint": "SameForAll",
+      "isRequired": false,
+      "isSearchable": false,
+      "inputHint": "SingleLine"
+    },
+    {
+      "name": "materialComposition",
+      "label": {
+        "de": "Zusammensetzung Material",
+        "en": "material composition"
+      },
+      "type": {
+        "name": "ltext"
+      },
+      "attributeConstraint": "None",
+      "isRequired": false,
+      "isSearchable": false,
+      "inputHint": "MultiLine"
+    }
+  ]
+}
+
+describe 'ProductTypeImporter', ->
```

Comment:
I guess those are integration tests. Can you put this file into an `spec/integration` folder?